.ansible-lint

```diff
@@ -0,0 +1,13 @@
+---
+# .ansible-lint
+
+exclude_paths:
+  - roles
+  - playbooks
```

Comment:
why does we exclude playbooks here?

Reply:
Because we will not change playbooks in the downstream controllers, only pull from upstream. And in the upstream controller there is no ci-tool to automatically start the linter.

Reply:
And ansible-lint will check availability of all roles included in playbooks. So in this case we will must download all roles in CI-pipeline.